src/components/Block/Actions.vue

```diff
@@ -5,7 +5,7 @@
       :key="plugin"
       @click="execute(plugin)"
       :loading="loading"
-      :disabled="!$auth.isAuthenticated"
+      :disabled="!$auth.isAuthenticated.value"
```

Comment:
Why do we need to use `.value` with $auth? It feel not intuitive. Can't we force it to not use .value?

Reply:
Because we are using ref() in the plugin to make it reactive. I'm not sure if there is a better way.

Reply:
We could use `unref(auth.provider)`

Reply:
I see, but we wont be able to use it inside a template? Not sure if it's better, the `.value` should be fine.

Reply:
How is this: https://github.com/balancer-labs/snapshot/pull/188/commits/11e1ec09f97882a347530a2945656fd61e92be17

Reply:
I think its better if we use the same approach everywhere to make it simpler to understand the code

Reply:
Ok